client/src/app/site/agenda/components/speaker-list/speaker-list.component.html

```diff
@@ -0,0 +1,116 @@
+<os-head-bar [nav]="false" [goBack]="true">
+    <!-- Title -->
+    <div class="title-slot">
+        <h2>
+            <span *ngIf="viewItem">{{ viewItem.getTitle() }}:</span> <span translate> List of speakers </span>
+        </h2>
+    </div>
+</os-head-bar>
+
+<mat-card class="speaker-card">
+    <!-- List of finished speakers -->
+    <mat-expansion-panel *ngIf="finishedSpeakers && finishedSpeakers.length > 0" class="finished-list">
+        <mat-expansion-panel-header>
+            <mat-panel-title translate> Last speakers </mat-panel-title>
+        </mat-expansion-panel-header>
+        <mat-list>
+            <!-- {Number}. {full_name} {time} minutes (Start time: {begin_time}) [close button] -->
+            <mat-list-item *ngFor="let speaker of finishedSpeakers; let number = index">
+                <div class="finished-prefix">
+                    <span>{{ number + 1 }}</span> <span>.&nbsp;{{ speaker }}</span>
+                </div>
+                <div class="finished-suffix">
+                    <!-- TODO: No Date or time class yet -->
+                    <!-- <span> calc time here &nbsp;</span> -->
+                    <!-- <span translate>minutes</span> -->
+                    <span>&nbsp;(</span> <span translate>Start time</span> <span>:&nbsp;{{ speaker.begin_time }})</span>
+                </div>
+                <button
+                    mat-stroked-button
+                    matTooltip="{{ 'Remove' | translate }}"
+                    *osPerms="'agenda.can_manage_list_of_speakers'"
+                    (click)="onDeleteButton(speaker)"
+                >
```

Comment:
Please put closing brackets in the same row as the last attribute. Same for every occurence..

Reply:
That did not work in my IDE somehow. I suppose there was a new package to automatically align HTML

Reply:
tried multiple times with other settings. Apparently prettify supports html now.